src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -858,7 +858,7 @@ def updateJobInformation(self, workflow, task, **kwargs):
             priority = (int(kwargs['requestPriority']) + int(kwargs['taskPriority'])*self.maxTaskPriority)
             try:
                 sd.edit('WMAgent_JobID =!= "UNDEFINED" && WMAgent_SubTaskName == %s && WMAgent_RequestName == %s'% (classad.quote(str(task)),classad.quote(str(workflow))),
-                        "JobPrio", classad.ExprTree('"%s"'% priority))
+                        "JobPrio", classad.ExprTree('"%d"'% priority))
```

Comment:
I think this is wrong.  I think it should be along the lines of:

classad.Literal(int(priority))

The current patch casts the priority to a string.